cmd/cloudwaste.go

```diff
@@ -16,9 +16,16 @@ func main() {
 		}
 	)
 
-	rootCmd.AddCommand(scan.Cmd())
+	l, err := zap.NewDevelopment()
+	if err != nil {
+		panic("failed to create logger")
+	}
+	defer l.Sync()
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Error return value of `l.Sync` is not checked (errcheck)